proj/src/main/webapp/script.js

```diff
@@ -50,17 +50,85 @@ function errorCallback(err){
 function getLocation() {
     if(navigator.geolocation){
         navigator.geolocation.getCurrentPosition(successCallback,errorCallback,{timeout:10000, enableHighAccuracy:false});
-    }
-    else{
+    } else{
         console.log("Error. Geolocation not supported or not enabled");
     }
     return;
 }
 
+/*
+    Setting password 
+*/ 
+
+// Check if the password meets the password requirement 
+$('#pass').on('keyup', function(){
+    const password = document.getElementById("pass");
+    var goodPassword = false;
+    passwordLength(password, goodPassword);
+    containsNumber(password, goodPassword);
+    containsSymbol(password, goodPassword);
+    // enables sign up button if the pw requirements are met
+    if (goodPassword === true) {
+        $('#btnSignUp').prop('disabled', false);
+    } else{
+        $('#btnSignUp').prop('disabled', true);
+    }
+});
+
+function passwordLength(password){
+    if (password.value.length > 7){
+        $('#pwLength').addClass('alert-success');
+        goodPassword = true; 
+    } else{
+        $('#pwLength').removeClass('alert-success');
+        goodPassword = false;
```

Comment:
goodPassword isn't defined in this method. It looks like it is supposed to be an input, but as I mentioned above, it might be better to return this value instead. Passing in a boolean for mutation seems a bit risky.

Reply:
gotcha, revising right now